pkg/3scale/amp/operator/redis_options_provider.go

```diff
@@ -306,3 +365,22 @@ func (r *RedisOptionsProvider) redisConfigMapResourceVersion() (string, error) {
 	}
 	return cm.GetResourceVersion(), nil
 }
+
+func (r *RedisOptionsProvider) backendRedisSecretLabels() map[string]string {
```

Comment:
are we doing watched by as part of this PR or is this a mistake?

Reply:
it's needed for testing. If it will be implamented different in "watched-by improvements" PR. So we will integrate it and will update, hope it's ok, it's not mistake.

Reply:
Removed. Thank you for comment!